spk/sickchill/Makefile

```diff
@@ -0,0 +1,41 @@
+SPK_NAME = sickchill
+SPK_VERS = v2020.11.24-1
```

Comment:
I am not use DSM package manager will properly detect "version increase" with such version pattern... let's try and see...

Reply:
Technically this should be the package version, not SC version. I think it should be whatever the date is of when the package is built/released because that is the version of SC it includes, but then after that when SC updates it's just the package version.

Reply:
@Nyaran can you drop the v in front of the version number?

Reply:
Sorry, hard week at work :(

Package updated to SickChill v2021.03.10-1 and now the SPK version is the build date.

BTW, I was trying to remove the "cross" part, but I received some errors. Is really possible to remove the cross part? I didn't find any example that could help me guess what I'm doing wrong

As usual... the package on first post is updated